lambdas/observe/observe_lambda_function.py

```diff
@@ -62,10 +71,40 @@ def lambda_handler(event, context):
             # Update the marker in DynamoDB
             data_service.update_marker(marker)
             logger.info(f"Successfully updated marker with ID {marker.get_marker_id()}.")
+
+            # Check notification condition and notify subscribers
+            if should_notify(marker):
+                emails = marker.get('subscribedEmails', [])
+                if emails:
+                    try:
+                        notification_service.notify_subscribers(marker.get_marker_id(), emails)
```

Comment:
@murauskas101 do 
```
notification = marker.get_name() + '\n' + marker.get_status()
notification_service.notify_subscribers(notification, emails)
```
 instead of `marker.get_marker_id()`